docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,238 @@
+
+```yaml
+piVersion: v1
+baseDomain: example.com
+controlPlane:
+  name: master
+  replicas: 3
+compute:
+- name: worker
+  replicas: 0
+metadata:
+  name: test-cluster
+platform:
+  openstack:
+    apiFloatingIP: 128.0.0.1
+    cloud: mycloud
+    defaultMachinePlatform:
+      type: m1.s2.xlarge
+    externalNetwork: external
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+```
+
+## Deploying an OpenShift Control Plane using UPI
+
+Deploying a control plane in UPI also requires little to no modifications from your normal install workflow. To avoid unnessary duplication, the steps to stand up an OpenShift control plane won't be detailed here. Instead, follow the [UPI install documentation](install_upi.md). Since workers are deployed manually in UPI, you can simply choose not to deploy compute nodes. If you are using our ansible scripts, to deploy only a control-plane simply skip running the `compute-nodes.yaml` script.
+
+## Preparing your OpenShift Cluster to use SR-IOV Networks
+
+Once your OpenShift control plane is running, you will need to install the SR-IOV Network Operator. To do so, you will need access to an account on your OpenShift cluster with `cluster-admin` privileges. Once you are logged into that account, follow [these steps](sriov-operator) to install the Operator.
+
+## Creating SR-IOV Networks for Compute Machines
+
+The SR-IOV networks that are attached to compute nodes need to be created before creating the nodes. Please reference the example of how to create radio and uplink provider networks in OpenStack.
+
+```sh
+# Create Networks
+openstack network create radio --provider-physical-network radio --provider-network-type vlan --provider-segment 120
+openstack network create uplink --provider-physical-network uplink --provider-network-type vlan --external
+
+# Create Subnets
+openstack subnet create --network radio --subnet-range <radio_network_subnet_range> radio
+openstack subnet create --network uplink --subnet-range <uplink_network_subnet_range> uplink
+```
+
+## Creating SR-IOV Compute Nodes in IPI
+
+TODO
+
+## Creating SR-IOV Compute Nodes in UPI
+
+Since we largely leave UPI implementation up to the customer, there is no official script for deploying SR-IOV compute nodes. However, we will use a verified example based on the [compute-nodes.yaml](../../../upi/openstack/compute-nodes.yaml) script to walk you through the process. To use this script, open up a terminal to the location of `inventory.yaml` and `common.yaml` UPI Ansible scrpits. In the example below, we add the provider networks named `radio` and `uplink` that we created before to the `inventory.yaml` file. Note that the count parameter specifies the number of virtual functions (VFs) to attach to each worker node.
```

Comment:
```suggestion
Since we largely leave UPI implementation up to the customer, there is no official script for deploying SR-IOV compute nodes. However, we will use a verified example based on the [compute-nodes.yaml](../../../upi/openstack/compute-nodes.yaml) script to walk you through the process. To use this script, open up a terminal to the location of `inventory.yaml` and `common.yaml` UPI Ansible scripts. In the example below, we add the provider networks named `radio` and `uplink` that we created before to the `inventory.yaml` file. Note that the count parameter specifies the number of virtual functions (VFs) to attach to each worker node.
```